.gitignore

```diff
@@ -11,6 +11,9 @@ build
 # TODO tasks
 todo
 
+# config
+config.toml
```

Comment:
I am not sure why we need `config.yml` here
As we have convention to use Environment Variables.